crowbar_framework/app/controllers/api/crowbar_controller.rb

```diff
@@ -19,6 +19,15 @@ class Api::CrowbarController < ApiController
 
   api :GET, "/api/crowbar", "Show the crowbar object"
   api_version "2.0"
+  example '
+  {
+    "version": "4.0",
```

Comment:
I agree with @santiph. Github PR comments should never be considered as documentation ...